app/uk/gov/hmrc/economiccrimelevyaccount/views/AccountView.scala.html

```diff
@@ -30,5 +34,101 @@
 
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-0">@messages("account.heading")</h1>
     <span class="govuk-caption-m">@messages("account.eclReferenceNumber", eclRegistrationReference)</span>
+    <p>
+
+
+        <br>
+
+    <div class="flex-container govuk-grid-row">
+    <!-- RETURNS CARD -->
+        <div class="card" id="aft-overdue-charges">
```

Comment:
I'd recommend making a templated component 'Card.scala.html` that takes parameters and you can re-use throughout this view, rather than duplicating the HTML, it will make it more readable and re-usable